workflow/common/common.go

```diff
@@ -57,6 +57,8 @@ const (
 	LabelKeyCronWorkflow = workflow.WorkflowFullName + "/cron-workflow"
 	// LabelKeyWorkflowTemplate is a label applied to Workflows that are submitted from Workflowtemplate
 	LabelKeyWorkflowTemplate = workflow.WorkflowFullName + "/workflow-template"
+	// LabelKeyWorkflowTemplate is a label applied to Workflows that are submitted from ClusterWorkflowtemplate
+	LabelKeyClusterWorkflowTemplate = workflow.WorkflowFullName + "/cluster-workflow-template"
```

Comment:
Why do we need a separate label here? Why would we consider `Workflows` submitted from a `WorkflowTempalte` and from a `ClusterWorkflowTemplate` differently?

Reply:
yes, It is from different resources.  WFT and CWFT can have same name. The workflow should have a separate label for workflowtemplate and clusterworkflowtemplate

Reply:
I thought the scope name of a WorkflowTemplate was replaced to be: `cluster/<template_name>` and `namespaced/<template_name>`. Would this be enough to make the distinction?

Reply:
Good point. we can have a common label for source  like `originatefrom` or `createfrom`
kind/name
`workflow-template/test`
`cron-workflow/test`
`cluster-workflow-template/test`
what is your thought?

Reply:
So we can extend for `clustercronworkflow` also in future

Reply:
Label value and Name has same legal character limitation `Valid label values must be 63 characters or less and must be empty or begin and end with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.`   label value can't add special character to separate the `namespace` and `name`.
I am more lean-to keep the separate label as the current implementation.